src/hat.cpp

```diff
@@ -560,9 +638,6 @@ void controlLed(bool isBeatDetected)
     }
     else
     {
-        EVERY_N_MILLIS(2)
-        {
-            fadeToBlackBy(leds, NUM_LEDS, 50);
-        }
+        fadeLeds(400);
```

Comment:
Why 400? :O 
Maximum brightness is 255.
If you just want to make it black you can use FastLED.clear();
Tbf using a 2 doing it every 2 ms would have a similar effect.